sdk/nodejs/runtime/closure/codePaths.ts

```diff
@@ -222,7 +222,7 @@ function findDependency(root: readPackageTree.Node | undefined | null, name: str
             const childFolderName = filepath.basename(child.path);
             const parentFolderName = filepath.basename(filepath.dirname(child.path));
             if (parentFolderName[0] === "@") {
-                childName = filepath.join(parentFolderName, childFolderName);
+                childName = `${parentFolderName}/${childFolderName}`;
```

Comment:
Minor - but wouldn't hurt to add a comment with a link to https://docs.npmjs.com/misc/scope describing the format so that we don't make mistakes like this again.